### ISSUE TITLE
Playing with phox v2

### ISSUE THREAD

**Issue by viewpointsa** [sdebby/RC_433_sig_analyze#1]
Hello,

I trying to clone my phox remote command 4-buttons:

![Capture d’écran 2024-10-21 113011](https://github.com/user-attachments/assets/d34b1c53-fa1a-4862-85c2-ecd240974bd2)

I have recorded this codes:
f8924xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx b6496492c965 8
f8924xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx b6496492c965 8
f8924xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx b6496492c965 8
f8924xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx b24b6492cb24 8
f8924xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 924b6db2db64 8
f8924xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 924b6db2db64 8
f8924xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 96496db2d925 8
f8924xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx 92db24b2c925 8

Do you find this algorithm to generate the rolling code ?

**Comment by sdebby** [comment]
In order to answer this you need to be more specific .
Are these codes generated from the same button pressing ?

**Comment by viewpointsa** [comment]
Yes, just the first button. But I wanted to know because I don't understand the phrase ‘breaking down the rolling code I found a pattern: the rolling codes will start with 0x92 or 0xb6 on both buttons.’, do you only need that for it to work?
